recipes/hangouts/webview.js

```diff
@@ -1,7 +1,8 @@
 module.exports = Ferdium => {
   const getMessages = () => {
     // get unread messages
-    const count = document.querySelector('#hangout-landing-chat iframe').contentWindow.document.querySelectorAll('.ee').length;
+    let count = 0;
+    for (const span of document.querySelectorAll('span[jsname=DW2nlb]'))  count += span.textContent == '' ? 0 : Number.parseInt(span.textContent);
```

Comment:
Please use `Ferdium.safeParseInt` which will remove the need for the ternary null/blank check.